### ISSUE TITLE
pip install sptag. Wheels for Windows 10 fails to import _SPTAG

### ISSUE THREAD

**Issue by EmElleE** [microsoft/SPTAG#317]
**Describe the bug**

@PhilipBAdams it seems like you are the only one working on this project.
Basically cannot access SPTAG through the python package

I think this has to do with python 3.8 and swig version that generates the interface needs to be updated. I would do it but I am not sure how... I would go about modifying the build system to use swig 4.0.1

currently it uses swig 3.0. x

https://docs.python.org/3/whatsnew/3.8.html#bpo-36085-whatsnew
```
from sys import version_info
if version_info >= (2, 7, 0):
    def swig_import_helper():
        import importlib
        pkg = __name__.rpartition('.')[0]
        mname = '.'.join((pkg, '_SPTAG')).lstrip('.') 
        return importlib.import_module(mname) <-- fails at this line here. 
    _SPTAG = swig_import_helper()
    del swig_import_helper
elif version_info >= (2, 6, 0):
    def swig_import_helper():
        from os.path import dirname
        import imp
        fp = None
        try:
            fp, pathname, description = imp.find_module('_SPTAG', [dirname(__file__)])
        except ImportError:
            import _SPTAG
            return _SPTAG
        if fp is not None:
            try:
                _mod = imp.load_module('_SPTAG', fp, pathname, description)
            finally:
                fp.close()
            return _mod
    _SPTAG = swig_import_helper()
    del swig_import_helper
else:
    import _SPTAG
del version_info

```
**To Reproduce**
Do a pip install sptag using python 3.8
or even build from source and import SPTAG from a test.py file.

**Expected behavior**
should import without dll not found error.

**Screenshots**

**Desktop (please complete the following information):**
Windows 10

**Comment by ArEnSc** [comment]
```
ImportError: DLL load failed while importing _SPTAG: The specified module could not be found.
```
I am getting the same thing.

**Comment by EmElleE** [comment]
I figured it out you have to force the code to use swig 4.02

**Comment by ArEnSc** [comment]
@jinwei don't recommend using this, it isn't at all worth the effort

**Comment by jinwei14** [comment]
> @Jinwei don't recommend using this, it isn't at all worth the effort

any recommendation of using the pypi package properly ?

**Comment by ArEnSc** [comment]
no its a complex build and the actual tech isn't documented, it's not really a developer product, use something like ... chromadb

**Comment by jinwei14** [comment]
I actually resolved it by using the Ubentu 20.04 on wsl, seems swig in the pypi pkg for windows is a bit outdated. I have asked the owner to update it, Thanks again for your help! @ArEnSc
